packages/runtime/container-runtime/src/opLifecycle/opSplitter.ts

```diff
@@ -152,10 +152,6 @@ export class OpSplitter {
 		);
```

Comment:
should name of this function (splitCompressedBatch) be changed?
I think it does not matter if it's compressed any more (as grouping produces an op that does not have compression metadate). Also, we might have non-compressed by grouped ops, I assume.

Reply:
[AB#4024](https://dev.azure.com/fluidframework/internal/_workitems/edit/4024)